### DISCUSSION TITLE
Missions hebdomadaires et fidélisation des nouveaux joueurs.

### DISCUSSION THREAD

**Discussion by Bombafox** [Crownicles/Crownicles#2521] [Nouvelles idées]
Bonjour,
Comme tout le monde le sait, il y a un manque cruel de joueurs sur db qui impacte de nombreuses choses comme les fights (du fait de l'écart presque incomblable entre ceux qui ont peu de gloire et ceux qui en ont beaucoup, qui cause un déséquilibre palpable lorsque ces petits joueurs parviennent à diminuer cet écart et perdent alors tous leurs combats face aux meilleurs).

Ainsi, comment rééquilibrer tout ça sans disproportion ?
Il s'agit de fidéliser les nouveaux joueurs et de les motiver à jouer.

On pourrait transformer le top gloire mais il y a déjà des récompenses, et faire une refonte complète prendrait énormément de temps.

On pourrait donc modifier le top week, car à part le badge du top 1 il n'y a aucune récompense.

Ma suggestion est donc de faire une "refonte" du tw en ajoutant des missions hebdomadaires pour avoir des récompenses comme de l'argent ou de l'xp (ressources cruciales pour les nouveaux joueurs) au reset du tw, en prenant en compte le niveau d'activité plutôt que le classement final des joueurs.
Voici quelques exemples de missions auxquelles j'ai pensé pour débloquer nos récompenses :
- Faire au minimum 3 missions (journalière, secondaire ou de campagne)
On pourrait ajouter un multiplicateur de bonus si on dépasse cet objectif.
Exemple : 
3 missions = bonus x1.0
4 missions = bonus x1.4
5 missions = bonus x1.5
Etc... Jusqu'à atteindre 10 missions qui rapporteraient le bonus maximum de x2.0

D'autres missions en plus de celle-ci (mais qui n'impacteraient pas le multiplicateur) pourraient être :
- Faire un minimum de 20 rapports
- Gagner 10 000 points

En ce qui concerne l'équilibrage, je ne m'y connais pas assez mais il faudrait que les récompenses d'argent et d'xp soient assez intéressantes en x1.0 sans que le total soit démesuré en x2.0.

**Comment by cocodu250** [comment]
un multiplicateur x2 est de toute façon démesuré, le mieux serait un x1.5 max
et donc dc l'atteindre en 5 missions

**Reply by Cirpacha** [reply]
5 missions c'est trop peu, peut-être gagner x0.1 à chaque fois qu'on fait 2 missions, comme ça le max reste à 10 missions avec un bonus de x1.5 tout au plus

**Reply by Bombafox** [reply]
Oui après c'est juste pour donner une idée dans l'ensemble, mais ça peut être modifié si c'est accepté.

(L'idée est surtout d'aider les nouveaux joueurs et de les garder actif.
Et retirer tout les gens qui n'ont fait que 1 rapport dans le classement serait pas mal aussi)

**Reply by cocodu250** [reply]
oublie pas que c'est 5 (ou 10) missions a partir des 3 missions qui rapportent le truc de base
